test/shared/pubsub_test.exs

```diff
@@ -40,7 +40,7 @@ defmodule Phoenix.PubSubTest do
     pid = spawn_pid
     assert Local.subscribers(config.local, "topic4") |> Dict.size == 0
     assert PubSub.subscribe(config.test, pid, "topic4")
-    assert Local.subscribers(config.local, "topic4") |> Enum.to_list == [pid]
+    assert Local.subscribers(config.local, "topic4") |> Enum.to_list == [{pid, nil}]
```

Comment:
Should we change subscribers to return the fastlanes? Maybe it should just return PIDs?